docs/src/app/api/[transport]/route.ts

```
@@ -0,0 +1,317 @@
+ID: ${page.url}
+`.trim()
+  )
+  .join("\n");
+
+const handler = createMcpHandler(
+  async (server) => {
+    server.tool(
+      "listAvailableDocs",
+      "Use this tool to learn about what Stack Auth is, available documentation, and see if you can use it for what you're working on. It returns a list of all available Stack Auth Documentation pages.",
+      {},
+      async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)